packages/components/releaseNotes/labkey/components.md

```diff
@@ -42,6 +42,10 @@ and add a tooltip for the AddControls with info on the data size limits.
     - Update to @labkey/api release version 0.0.29
     - Issue 39359: fix to prevent API call to study-reports-getReportInfos if we know study module isn't available
 
+### version TBD
+*Released*: 16 January 2020
+* Changes to include server side warnings while saving domain
```

Comment:
Looks like you need to move your release notes to the top of this file (maybe just got merged in the wrong direction).